java/src/main/java/com/google/oak/remote_attestation/InsecureAttestationVerifier.java

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.remote_attestation;
+
+import com.google.oak.util.Result;
+import oak.session.noninteractive.v1.AttestationEndorsement;
+import oak.session.noninteractive.v1.AttestationEvidence;
+
+/**
+ * A test verifier implementation that doesn't verify attestation evidence and is used for testing.
+ */
+public class InsecureAttestationVerifier implements AttestationVerifier {
+  /**
+   * Verify that the provided evidence was endorsed and contains specified reference values.
```

Comment:
This javadoc is misleading, and can be simplified, since only one value is returned.

Reply:
Updated